app/views/shared/_spam_honeypot.html.erb

```diff
@@ -0,0 +1,5 @@
+<%# Honeypot field intended to reduce spam %>
+<div class="visually-hidden" aria-hidden="true">
+  <label for="val">Ignore if you're not a robot</label>
+  <input type="text" tabindex="-1" autocomplete="off" id="val" name="<%= "#{form_name}" %>[val]"/>
```

Comment:
I believe this is so when tabbing through the form a user could not tab to this hidden input?